CHANGELOG.md

```diff
@@ -3,8 +3,20 @@
 **SDK**
 - **Upgraded:** Target SDK is now API 31
 - **Upgraded:** SDK dependencies to new versions
+    - androidx.core:core-ktx:1.7.0
+    - androidx.webkit:webkit:1.4.0
+    - androidx.security:security-crypto:1.1.4-alpha03
```

Comment:
Sorry, I forgot to mention that this one has to stay at `1.1.0-alpha01` due to compatibility issues.